lib/repository/artemis_client.dart

```diff
@@ -24,7 +24,7 @@ class ArtemisClientCreator {
     builder.subject = userId;
     builder.expiresAt = DateTime.now().add(Duration(days: 30));
     builder.issuer = 'www.example.com';
-    builder.setClaim('custom:tenant', tenantId);
+    builder.setClaim('https://platform.yonomi.cloud/tenant', tenantId);
```

Comment:
Does this belong in this commit?  Seems out of context.

Reply:
Yes but right now the bigger context is semver and this smaller context also needs to go. Adding various other merges is only going to make it more confusing IMO